design/ceph/ceph-external-cluster.md

```diff
@@ -157,3 +157,41 @@ monitoring:
 4. Create a new metric Service
 
 5. Create an Endpoint resource based out of the IP addresses either discovered or provided by the user in the script
+
+## Steps for creating external rook-ceph cluster
```

Comment:
This is intended to be user documentation, right? If so, this should be added under Documentation instead of the design docs. Should this be a new section in the [external cluster doc](https://rook.github.io/docs/rook/v1.8/ceph-cluster-crd.html#external-cluster), or does that need to be modified as well?

Reply:
Yes this need modification.